Dockerfile

```diff
@@ -34,6 +35,7 @@ RUN set -x \
 	&& rm -rf "$GOPATH"
 
 RUN set -x \
+    && export GO111MODULE=off \
```

Comment:
A nit: Could you indent the new lines the same way as the rest, please?

Reply:
Oh woops, my mistake, yep I got it...